aten/src/ATen/native/native_functions.yaml

```diff
@@ -8322,14 +8322,15 @@
     CompositeExplicitAutograd: elu_
 
 - func: glu.out(Tensor self, int dim=-1, *, Tensor(a!) out) -> Tensor(a!)
+  structured: True
```

Comment:
Do we need "structured_inherits: TensorIteratorBase" here?

Reply:
i'm not sure but currently it's not possible due to

[`DECLARE_DISPATCH(activation_fn, glu_stub);`](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/Activation.h#L54) and [`using activation_fn = void (*)(TensorIterator&);
`](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/Activation.h#L16) in https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/Activation.h